### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by ljte**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,9 +11,10 @@ services:
 
 before_script:
   - psql -c "CREATE DATABASE test;" -U postgres
+  - python run.py &
 
 script:
-  - pylint --disable=no-self-use,too-few-public-methods,no-member,broad-except --max-line-length=120 department_app/
+  - pylint --disable=no-self-use,too-few-public-methods,no-member,broad-except,eval-used --max-line-length=120 department_app/
```
Comment (inauros) [inline .travis.yml]: Please remove eval-used from the --disable list and fix your code. The eval function is unsafe and it is not recommended to use it unless you really need it.

File: department_app/service/utils.py [hunk 2]
```diff
@@ -96,3 +98,61 @@ def update_record(model, record, **new_fields):
...
+def filter_employees_by_date_period(start_date: date, end_date: date):
+    """return employees with the birthday inside the period from start_date to end_date
+    """
+    if not isinstance(start_date, date) or not isinstance(end_date, date):
+        raise ValueError('invalid date format')
+    emps = Employee.query.filter(Employee.bday >= start_date).all()
+    return [emp for emp in emps if emp.bday <= end_date]
+
+
+def init_db_for_test():
```
Comment (inauros) [inline department_app/service/utils.py]: This function should be moved to a python file under the tests package. All code somehow related to tests should be moved to the tests pacakge.

**Review by inauros — CHANGES_REQUESTED** [review]
In addition to the comments below, update your unit tests to cover the missing lines of code, since the missing lines of code are not non-testable code, they can be covered by unit tests.

Also I noticed that you are running the development server as part of unit testing. Unit tests should not require the development server to run. It looks like you are doing something wrong here. Please fix it.

File: .travis.yml (hunk 1 above)
Comment (ljte) [reply .travis.yml]: done

File: department_app/service/utils.py (hunk 2 above)
Comment (ljte) [reply department_app/service/utils.py]: done

**Pull request merged**